js/portfolio/articles.js

```diff
@@ -0,0 +1,41 @@
+var articles = [];
+
+function Article (opts) {
+  // TODO: Use the js object passed in to complete this contructor function:
```

Comment:
Are you making an article here? Is this set of data what's best for your needs, or would you be better served by something else?